src/main/java/com/venom/CRUD/CrudApplication.java

```diff
@@ -0,0 +1,22 @@
+package com.venom.CRUD;
+
```

Comment:
package names are not supposed to be in caps, 
put something like `com.venom.employee`